rpc-test/tests/nonblocking.rs

```diff
@@ -0,0 +1,41 @@
+use {
+    solana_client::{nonblocking::tpu_client::TpuClient, tpu_client::TpuClientConfig},
+    solana_sdk::{pubkey::Pubkey, system_transaction},
+    solana_test_validator::TestValidatorGenesis,
+    std::{
+        sync::Arc,
+        time::{Duration, Instant},
+    },
+};
+
+#[tokio::test]
+async fn test_tpu_send_transaction() {
```

Comment:
There aren't too many unit tests for the tpu client.  Since I was in the area, I ran the token cli bench against testnet to make sure it works.  Once this is validated, I'd like to add another to be sure that the underlying leader cache is properly updated.